src/main/java/io/github/moulberry/notenoughupdates/NEUEventListener.java

```diff
@@ -240,7 +243,7 @@ public void onTick(TickEvent.ClientTickEvent event) {
             inventoryLoadedTicks = 3;
         }
 
-        if (Keyboard.isKeyDown(Keyboard.KEY_NUMPAD1) && Keyboard.isKeyDown(Keyboard.KEY_NUMPAD4) && Keyboard.isKeyDown(Keyboard.KEY_NUMPAD9)) {
+        if ((Keyboard.isKeyDown(Keyboard.KEY_NUMPAD1) && Keyboard.isKeyDown(Keyboard.KEY_NUMPAD4) && Keyboard.isKeyDown(Keyboard.KEY_NUMPAD9) || NotEnoughUpdates.INSTANCE.config.hidden.bHopBan)) {
```

Comment:
please don't add something called bhop, there will be some non on the forums complaining about it